src/views/AmalgamationRegular/BusinessInfo.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div id="amalgamation-regular-business-info">
```

Comment:
ID = folder name + component name. Here I used the long names. In the routes I used abbreviations (amalg + reg).

The component name itself (line 81) is the CamelCase version of the ID (consistent with other components).